stratum/hal/lib/tdi/dpdk/dpdk_sde_wrapper.cc

```diff
@@ -142,13 +143,13 @@ ::util::Status TdiSdeWrapper::HotplugPort(
             << " qemu_vm_chardev_id=" << hotplug_attrs->qemu_vm_chardev_id
             << " qemu_vm_device_id=" << hotplug_attrs->qemu_vm_device_id
             << " native_socket_path=" << hotplug_attrs->native_socket_path
-            << " qemu_hotplug = " << hotplug_attrs->qemu_hotplug;
+            << " qemu_hotplug_mode = " << hotplug_attrs->qemu_hotplug;
```

Comment:
As per hotplug_attrs->qemu_hotplug = (hotplug_config.qemu_hotplug_mode != 0), hotplug_attrs->qemu_hotplug will be true in case of ADD/Del and on line 146 instead of printing mode ADD/DEL, we will be printing true

Reply:
Since stratum makes the decision of call appropriate hotplug API, bf_pal_hotplug_add for HOTPLUG_MODE_ADD and bf_pal_hotplug_del for HOTPLUG_MODE_ADD, maybe we can remove qemu_hotplug parameter from hotplug_attributes_t structure

Reply:
Good point. Something to consider doing after this release.